### DISCUSSION TITLE
World Coord Info

### DISCUSSION THREAD

**Discussion by marcoma9023** [pelicanmapping/osgearth#1857] [Support Q&A] [answered]
Hi,

![Immagine 2021-11-10 144844](https://user-images.githubusercontent.com/87537804/141124782-83bb87e9-ab4f-42a3-af40-852507761fa3.png)

As you can see from the image, I putted three point: center, sx, dx on google maps to help you to unrderstand my problem.

Then, for the point "center" I created a gepoint with a specific lat, lon. Since I need the world coordinate, I transformed by: 

`osg::Vec3d centerWorld;
osgEarth::GeoPoint point(lonCenter, latCenter, osgEarth::ALTMODE_ABSOLUTE);
point.toWorld(centerWorld);`

Then, for the geopoint "sx" and "dx" I transformed in the same way....
 `osg::Vec3d dxWorld;
osgEarth::GeoPoint dx_point(lonDx, latDx, osgEarth::ALTMODE_ABSOLUTE);
dx_point.toWorld(dxWorld);`

The problem is that, I expect the x of the world point "dx" to be greater than the world coordinate of the center.

Instead it is not so!

Where am I wrong? Can you tell me something about the world coordinate system?

**Comment by jasonbeverage** [comment, marked as answer]
osgEarth uses a 3D geocentric world coordinate system, described [here](https://en.wikipedia.org/wiki/Earth-centered,_Earth-fixed_coordinate_system).  Basically it's a 3D cartesian system with the center of the earth being the origin.

**Reply by marcoma9023** [reply]
Thanks for reply, but I can't understand something:

I have geopoint 
`geoGenter(17.2191, 40.4771)` 
`geoSx(17.2188, 40.4785)` 
`geoDx(17.2193, 40.4785)` 

in world (ecef) coordinates:
CENTER:
X : 4640.728   km
Y : 1438.241   km
Z : 4118.426   km

SX:
X : 4640.639   km
Y : 1438.186   km
Z : 4118.544   km

DX:
X : 4640.626   km
Y : 1438.227   km
Z : 4118.544   km

why for example, for the x, the value of P2 is lesser than center?

**Comment by jasonbeverage** [comment]
It's because the X value is along a fixed axis from the center of the earth and not based on your center point.
